tests/unit/aofrw.tcl

```diff
@@ -58,10 +58,9 @@ start_server {tags {"aofrw external:skip"}} {
 }
 
 start_server {tags {"aofrw external:skip"} overrides {aof-use-rdb-preamble no}} {
+    r config set appendonly yes
+    waitForBgrewriteaof r
```

Comment:
here too, let's move the configuration to the server startup and no need to revert it at the end. (external:skip)